src/js/touchevents.tsx

```diff
@@ -27,6 +27,7 @@ export type TouchEventBoundaryProps = {
    * @deprecated
    */
   ignoredDisplayNames?: Array<string | RegExp>;
+  children?: React.ReactNode;
```

Comment:
```suggestion

  children?: React.ReactNode;
```

I'd also add method docs.

Reply:
ok